Makefile

```diff
@@ -60,8 +60,7 @@ production-requirements:
 	pip install -qr requirements.txt --exists-action w
 
 test: clean test_requirements
-	coverage run ./manage.py test designer --settings=designer.settings.test
-	coverage report
+	pytest
```

Comment:
Not that we don't already have enough warnings to fix here, but using `python -Wd -m pytest` instead might turn up a few more.